src/index.ts

```diff
@@ -11,12 +11,19 @@ export * from './SanityClient'
 export * from './types'
 
 // Set the http client to use for requests, and its environment specific middleware
-const httpRequest = defineHttpRequest(envMiddleware)
+const httpRequest = defineHttpRequest(envMiddleware, {})
 /** @public */
 export const requester = httpRequest.defaultRequester
```

Comment:
Out of scope for this PR, but it does seem like we should export `defineHttpRequest` instead of `requester`. I'm not sure what use there is for `requester` anymore. What do you think @rexxars?

Reply:
There are a few usages internally, but I'm sure they could be done differently: https://github.com/search?q=org%3Asanity-io+requester&type=code